docs/source/supergraphs.md

```diff
@@ -74,3 +74,16 @@ rover supergraph compose --config ./supergraph.yaml > prod-schema.graphql
 ```
 
 > For more on passing values via `stdout`, see [Using `stdout`](./conventions#using-stdout).
+
+#### Gateway compatibility
+
+The `rover supergraph compose` command produces a supergraph by utilizing the composition functions of the [`@apollo/federation`](https://www.apollographql.com/docs/federation/api/apollo-federation/) package.  As that library is still in pre-1.0 releases (as are Rover and Apollo Gateway), some updates to Rover may result in a supergraph with new functionality and therefore necessitate complimentary updates to Apollo Gateway.
+
+Apollo Gateway will fail to start-up if it's provided with a supergraph that it doesn't support, so it's recommended to test launching Apollo Gateway with the supergraph it will be ultimately be deployed with in a CI pipeline to ensure compatible versions.
```

Comment:
"will fail" -> "fails"

"start-up" -> "start up"

"supergraph" -> "supergraph schema that it doesn't support." <- end sentence

"so it's recommended" -> "To ensure compatibility, we recommend that you test launching your gateway in a CI pipeline with the supergraph schema it will ultimately use in production."